### PR TITLE
cli(ticdc): refactor cdc cli changefeed cyclic to use api v2

### LINKED ISSUE #5812: TiCDC CLI: cdc cli changefeed cyclic use open api v2
Refactor the cli changefeed cyclic to use TiCDC Open API v2

### PR DESCRIPTION
**Pull request by crelax**
### What problem does this PR solve?

Issue Number: close #5812

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
refactor cdc cli changefeed cyclic to use api v2
```

### CODE REVIEW

File: pkg/api/v2/verify_table.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+type VerifyTableInterface interface {
+	Verify(ctx context.Context, verifyTableConfig *v2.VerifyTableConfig) (*v2.Tables, error)
+}
+
+// verifyTable implements VerifyTableInterface
+type verifyTable struct {
+	client rest.CDCRESTInterface
+}
+
+// newUnsafe returns unsafe
```
Comment (zhaoxinyu) [inline pkg/api/v2/verify_table.go:38]:
```suggestion
// newVerifyTable returns verifyTable
```

**Comment by zhaoxinyu** [comment]
@crelax, jenkins-ticdc/verify failed. Please execute `make check` to solve the lint problems.

**Comment by crelax** [comment]
> @crelax, jenkins-ticdc/verify failed. Please execute `make check` to solve the lint problems.

Hi xinyu, this pr is not going to be merged for now, since it relies on the refraction of other changefeed cli. It will be tested and processed to merge then

**Comment by crelax** [comment]
cli no longer supports changefeed cyclic

**Pull request closed without merging**